.github/workflows/python_lint.yaml

```
@@ -17,10 +17,10 @@ jobs:
       - name: Checkout source repository
         uses: actions/checkout@v4
 
-      - name: Set up Python environment 3.8
+      - name: Set up Python environment 3.12
```

```suggestion
      - name: Set up Python
```

shorter name == better name 😎 

I want to get out of specifying the version in the `name` to reduce number of lines that would ever need changes on future update..